lib/mock.js

```diff
@@ -1,5 +1,7 @@
 const Module = require('module')
 const { isAbsolute } = require('path')
+const process = require('./process.js')
+const { platform } = process
```

Comment:
As before it seems like we could just keep using process.platform in the rest of the code but not a blocker.